.github/workflows/sultan-ci.yml

```diff
@@ -0,0 +1,22 @@
+on: push
```

Comment:
If you case you've forgot to run it on `pull_request`:

```suggestion
on:
  pull_request:
  push:
    branches:
      - juniper
```

Reply:
Triggering on `pull_request` won't allow us to fetch the branch name. Push actually works on merge too. Here's a proof:
- The workflow run on devstack repo: https://github.com/appsembler/devstack/runs/3036630625?check_suite_focus=true
- The workflow run on Sultan: https://github.com/appsembler/sultan/runs/3036632015?check_suite_focus=true
- And here's a picture of the created image
<img width="1144" alt="Screen Shot 2021-07-10 at 9 54 21 AM" src="https://user-images.githubusercontent.com/11036472/125170660-d0250b80-e164-11eb-899a-0d35120e574e.png">